AlphaWallet/Transfer/Controllers/TransactionConfigurator.swift

```diff
@@ -101,7 +106,11 @@ class TransactionConfigurator {
                     if limit == BigInt(21000) {
                         return limit
                     }
-                    return limit + (limit * 20 / 100)
+                    if limit > GasLimitConfiguration.maxGasLimit {
```

Comment:
We need to calculate the +20% before comparing. Because if `limit` is approximately > 80% of `GasLimitConfiguration.maxGasLimit`, it'll run `else` and compute a value bigger than GasLimitConfiguration.maxGasLimit.

Reply:
Can use `min()` here too.